src/window_savefile.cpp

```diff
@@ -102,9 +111,9 @@ void Window_SaveFile::Refresh() {
 		return;
 	}
 
-	std::stringstream out;
+	out.str("");
 	if (override_index > 0) {
-		out << lcf::Data::terms.file << std::setw(2) << std::setfill(' ') << override_index;
+		out << lcf::Data::terms.file << std::setw(3) << std::setfill(' ') << override_index;
```

Comment:
btw this stringstream stuff is old legacy code. If you want you can replace it with appropriate fmt::format calls

Reply:
There are even more spots which use the stringstreams but I better not touch them because of the risk to break the `override_name` stuff in the savegames.